manager/src/main/resources/schemas/actions/SendToBridge.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "object",
+  "properties": {
+    "bridgeId": {
+      "type": "string",
+      "title": "Bridge ID",
+      "description": "The bridgeId of the instance to target.",
+      "example": "f1fbd010-93cf-4be1-aa78-b37ba48858fe"
+    }
+  },
+  "required": [
+    "bridgeId"
```

Comment:
as far as I remember this is not mandatory (even if we are discussing to make it mandatory)

Reply:
@danielezonca if it's not mandatory how can the user specify the target?

Reply:
Found out by default it's the bridge that is already attached to the processor. Fixed